maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -86,15 +96,15 @@ fun rememberMarkerDragState(): MarkerDragState = remember {
  * @param title the title for the marker
  * @param visible the visibility of the marker
  * @param zIndex the z-index of the marker
- * @param markerDragState a [MarkerDragState] to be used for observing marker drag events
  * @param onClick a lambda invoked when the marker is clicked
  * @param onInfoWindowClick a lambda invoked when the marker's info window is clicked
  * @param onInfoWindowClose a lambda invoked when the marker's info window is closed
  * @param onInfoWindowLongClick a lambda invoked when the marker's info window is long clicked
+ * @param onMarkerDrag a lambda invoked while the marker is dragged
  */
 @Composable
 fun Marker(
-    position: LatLng,
+    positionState: MarkerPositionState = rememberMarkerPositionState(),
```

Comment:
I replaced raw position with position state completely. But should I remain position parameter or make overload?

Reply:
An overload would be ideal, but I think it would be a maintenance burden to support polymorphic types for `position` and `positionState`. So, I'm in favor of replacing `position` and `markerDragState` here.